test/dotnet/ExpectedOutput/insertBulkErrors.out

```diff
@@ -0,0 +1,376 @@
+#Q#Create table sourceTable(a int, b int not null)
```

Comment:
can we integrate heap AM check in test framework?

Reply:
The tests to enable seq/enable index scan do the same check right? If we need it, going ahead in the next release I think we can do that as well